app/javascript/components/Articles/ArticlesDetails.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+
+type ArticleDetailsProps = {
+    title?:string,
+    content?:string,
+    user?:string,
+    created_at?:string
+}
+
+class ArticleDetails extends React.Component<ArticleDetailsProps> {
```

Comment:
This could be a React functional component, as it only contains a render method

Reply:
Just so confirm, you recommend using a functional component instead of a component class as the class is very simple?

Reply:
Yep, pretty much. That and functional components are faster and less resource intensive from memory.